index.md

```diff
@@ -3,20 +3,27 @@ layout: lesson
 root: ../..
 title: Programming with MATLAB
 ---
-The best way to learn how to program is to do something useful,
-so this introduction to MATLAB is built around a common scientific task:
-data analysis.
-Our real goal isn't to teach you MATLAB,
-but to teach you the basic concepts that all programming depends on. We use 
-MATLAB in our lessons because:
+The best way to learn how to program is to do something useful, so this
+introduction to MATLAB is built around a common scientific task: data analysis.
+Our real goal isn't to teach you MATLAB, but to teach you the basic concepts
+that all programming depends on. We use MATLAB in our lessons because:
 
 1. we have to use *something* for examples;
 2. it's well-documented;
 3. it has a large (and growing) user base among scientists in academia and the industry; and
 4. it has a large library of packages available for performing diverse tasks.
 
-But the two most important things are to use whatever language your colleagues are using, 
-so that you can share you work with them easily, and to use that language *well*.
+But the two most important things are to use whatever language your colleagues
+are using, so that you can share you work with them easily, and to use that
+language *well*.
+
+> ## Prerquisites
+>
+> Learners should have completed introductory lessons on:
+>
+> - the Unix shell (Files and Directories, Creating Things, Loops)
```

Comment:
Same question as @r-gaia-cs - what in the lesson requires shell knowledge? Particularly if learners are doing everything from inside the MATLAB IDE?
Thanks,
Greg